mlflow/models/python_api.py

```diff
@@ -96,7 +96,7 @@ def predict(
     input_path: Optional[str] = None,
     content_type: str = _CONTENT_TYPE_JSON,
     output_path: Optional[str] = None,
-    env_manager: _EnvManager = _EnvManager.VIRTUALENV,
+    env_manager: str = _EnvManager.VIRTUALENV,
```

Comment:
Can we wrap the pydoc args for this entry below with " " around the enumerated options to provide a visual cue to their allowable value entries? i.e.:

```python
        env_manager: Specify a way to create an environment for MLmodel inference:

            - "virtualenv" (default): use virtualenv (and pyenv for Python version management)
            - "local": use the local environment
            - "conda": use conda
```